service_user.go

```diff
@@ -27,14 +31,20 @@ type (
 	// CreateServiceUserRequest are the parameters required to create a
 	// ServiceUser.
 	CreateServiceUserRequest struct {
-		Username string `json:"username"`
+		Username           string   `json:"username"`
+		RedisACLCategories []string `json:"redis_acl_categories,omitempty"`
```

Comment:
Doesn't look like these keys are placed correctly. They should be under `access_control`. Also when the `access_control` key is present it must contain all three lists.